### ISSUE TITLE
transfer problem: bigger amount 50000 NEON

### ISSUE THREAD

**Issue by hypersoft2014** [neonlabsorg/neon-client-transfer#89]
When I try to transfer bigger amount 50000 NEON from Neon EVM DevNet (in Metamask wallet, address: 0xE10C3269D027c8Ed1d1FD6f51796D54e48455d34) to Solana DevNet (in Phantom wallet), the error shows that, gas fees becomes extremely too much high (97056947.961222 NEON) and cannot continue the transaction.

![screenshot](https://user-images.githubusercontent.com/8074368/167213836-f6520c79-e9c9-43d2-9777-4ce0055a4aa4.jpg)

**Comment by ThannDev** [comment]
Talk to Solana Support on the [Live support page](https://sol-community.com/) so that your issue can be looked into.

**Comment by ThomasDev62** [comment]
Talk to a Solana Support on the [Live support page](https://sol-community.com/) so that your issue can be looked into.

**Comment by hypersoft2014** [comment]
No, I think that’s program problem so I ca do nothing there, but I changed
to 40000 NEON then it’s successful.

**Comment by ThomasDev62** [comment]
Talk to the live support they’ll get you a solution to it
